optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -529,6 +549,64 @@ private void writeBestScorePerTimeSpentSummaryChart() {
         }
     }
 
+    private void writeSubSingleBenchmarkScoreCharts() {
```

Comment:
Had a similar thought process when I wrote this, and decided to leave it here.

Not sure about this chart being obsoleted by PLANNER-443, as that animal is a bit different from my point of view.